src/common/sendpacket.c

```diff
@@ -237,7 +260,10 @@ sendpacket(sendpacket_t *sp, const u_char *data, size_t len, struct pcap_pkthdr
     static const size_t buffer_payload_size = sizeof(buffer) + sizeof(struct pcap_pkthdr);
 
     assert(sp);
+    #ifndef HAVE_LIBXDP
```

Comment:
this modification will be reverted, please disregard it. I don't need the include guard anymore here